README.md

```diff
@@ -36,16 +36,22 @@ In case you don't want to set an environment variable, you can configure [vscode
 After you have written code, you can test it locally using `pytest`.
 To test a file, just run:
 
-```bash
-pytest CryCollege/week3/weierstrass_curve.py 
-```
+```bashpytest CryCollege/week3/weierstrass_curve.py``` 
```

Comment:
There is a missing newline. This won't render properly. Add a newline after `bash`.